.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Enhancement Request
+about: I'd like a feature to be implemented
+labels: enhancement
```

Comment:
I think the "enhancement" label is redundant. With your templates, every issue will have either "bug" or "enhancement", never both. So it's enough to just have "bug", and no label for enhancements.
It's also easier to change your mind later when there is just one label ("bug"). If here are two labels (bug and enhancement) and you decide that an enhancement is really a bug, you need to do two changes (drop the enhancement label, add the bug label) instead of one.

Reply:
There are also issues that have neither. 

It's also more natural to select all enhancements, than all non-bugs.

Reply:
With your templates, there will be no issue that has neither. All your templates have either one or the other.

Reply:
Yes. That's the goal. So if you see an issue with no label, you have don't have to wonder whether it's uncategorized, or an enhancement. You know it's an uncategorized.

Reply:
No, after your patch you will never see an issue which is "uncategorized", it will always have one of these two. Even if the bug reporter doesn't really know what's the difference, so he just picked one of the two at random. So now if I see the label "enhancement", I have to wonder even harder (and every time, not just once) whether this is really an "enhancement", because this label might have been set by a random user who knows nothing about our development process.
Anyway, I'm not a fan of the new templates, but if you are, and find another fan to commit it, I won't fight it either :-)

Reply:
We will still see uncategorized issues, because many still exist.